internal/ext/process/gen.go

```diff
@@ -37,6 +37,9 @@ func (r Runner) Generate(ctx context.Context, req *plugin.CodeGenRequest) (*plug
 		fmt.Sprintf("SQLC_VERSION=%s", req.SqlcVersion),
 	}
 	for _, key := range r.Env {
+		if key == "SQLC_AUTH_TOKEN" {
+			continue
+		}
```

Comment:
We don't really need this, it just seemed like a reasonable thing to do (prevent users from sending their auth tokens to plugins).